### PR TITLE
changed file structure of bamPEFragmentSize and added documentation

### PR DESCRIPTION
**Pull request by fidelram**

### CODE REVIEW

**Comment by dpryan79** [comment]
LGTM

File: deeptools/bamPEFragmentSize.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    print "Min.: {}\n1st Qu.: {}\nMean: {}\nMedian: {}\n" \
+          "3rd Qu.: {}\nMax.: {}\nStd: {}".format(fragment_len_dict['min'],
+                                                  fragment_len_dict['qtile25'],
+                                                  fragment_len_dict['mean'],
+                                                  fragment_len_dict['median'],
+                                                  fragment_len_dict['qtile75'],
+                                                  fragment_len_dict['max'],
+                                                  fragment_len_dict['std'])
+
+    print "\nRead lengths:"
```
Comment (bgruening) [inline deeptools/bamPEFragmentSize.py]: We should use logging here, right?

Comment (dpryan79) [inline deeptools/bamPEFragmentSize.py]: That's why the unix gods created `>` :)

Comment (bgruening) [inline deeptools/bamPEFragmentSize.py]: ;) NoNo!

Comment (fidelram) [inline deeptools/bamPEFragmentSize.py]: Why would logging improve in this case? Do you need the output in certain way for galaxy?

**Pull request merged**